src/modules/hotkeys/register.json

```diff
@@ -1,7 +1 @@
-{
-    "dev": true,
-    "github": 15,
-    "location": ".*",
-    "noapp": false,
-    "settings": true
-}
\ No newline at end of file
+{ "github": 15, "location": ".*", "noapp": false, "settings": true }
```

Comment:
New linter?

Reply:
no but due to removal of the `dev` property, prettier decided that it is fine to put it all in one line.